crates/napi/src/threadsafe_function.rs

```diff
@@ -328,18 +336,20 @@ impl<T: 'static, ES: ErrorStrategy::T> Drop for ThreadsafeFunction<T, ES> {
 
 unsafe extern "C" fn cleanup_cb(cleanup_data: *mut c_void) {
   let aborted = unsafe { Arc::<AtomicBool>::from_raw(cleanup_data.cast()) };
-  aborted.store(true, Ordering::SeqCst);
+  aborted.store(true, Ordering::Release);
 }
 
 unsafe extern "C" fn thread_finalize_cb<T: 'static, V: ToNapiValue, R>(
   _raw_env: sys::napi_env,
   finalize_data: *mut c_void,
-  _finalize_hint: *mut c_void,
+  finalize_hint: *mut c_void,
 ) where
   R: 'static + Send + FnMut(ThreadSafeCallContext<T>) -> Result<Vec<V>>,
 {
   // cleanup
   drop(unsafe { Box::<R>::from_raw(finalize_data.cast()) });
```

Comment:
`finalize_data` is no longer points to `Box<callback>`, so I think threadsafe function context is now leaking.

Reply:
No, `finalize_data` is still pointed to `Box<Callback>`, this PR just change `finalize_hint` from `Box<Callback>` to `aborted_ptr`

Reply:
Ah, sorry, my bad